templates/ui-frameworks/vue/web-app/ui/src/App.vue.hbs

```diff
@@ -1,19 +1,21 @@
 <template>
   <ClientProvider>
     <div>
-      <a href="https://developer.holochain.org/get-started/" target="_blank">
-        <img src="./assets/holochainLogo.svg" class="logo holochain" alt="holochain logo" />
-      </a>
-    </div>
-    <h1>Holochain Vue hApp</h1>
-    <div>
-      <div class="card">
-        <p v-if="loading">connecting...</p>
-        <p v-else-if="error">{{ error.message }}</p>
-        <p v-else>Client is connected.</p>
+      <div>
```

Comment:
Same here, just added a wrapper div.

Reply:
Thanks for fixing this. Curious to know if other templates exhibit the same issue.

Reply:
I saw it in Vue and Svelte (which I both changed), Lit was fine and for React I'm not familiar with the syntax, it might have the same problem let me check.

Reply:
React is fine, tested it right now and vanilla doesn't have any content anyway.